xt_training/utils/training.py

```diff
@@ -51,11 +51,8 @@ def train(args):
         on_exit=on_exit,
         use_nni=use_nni
     )
-    
-    # Save config file
-    if isinstance(config_path, str):
-        shutil.copy(config_path, f'{save_dir}/config.py')
-    else:
-        shutil.copy(config['__file__'], f'{save_dir}/config.py')
+
+    # Save config file in checkpoint directory
+    _save_config(save_dir, config_path)
```

Comment:
Hi Tim, so are we no longer wanting to save the git repo state in the Command Line API? Only the config?

Reply:
We are, but it now happens in the functional calls since we can do it always, not just for the CLI API